### ISSUE TITLE
Maven search command

### ISSUE THREAD

**Issue by reibitto** [reibitto/command-center#15]
To be able to quickly pull the latest version of a dependency into your clipboard and so on. Might be nice to also have a view that shows the transitive dependencies.

**Comment by tuleism** [comment]
TODO after #34:

- Support more input type, e.g CLI with `argsPreview` to search with `-g group -a artifact` and so on. Probably best to do after #39, otherwise we may send HTTP request multiple times.
- Support `%%` or `%` style query. Example:
  - "zio.dev" %% "zio" will expand to `g:"zio.dev" a:"zio_2.12"`. In this case, the scala version can either be fixed to some newer version.
  - "ch.qos.logback"  % "logback-classic" will expand to `g:"ch.qos.logback" a:"logback-classic"`.

**Comment by reibitto** [comment]
I'm closing this as I made changes to the search command recently. I'll create a new issue for other possible improvements.
